build

```diff
@@ -16,7 +16,19 @@ eval $(go env)
 # set default stage1 usr style and validate potential override
 RKT_STAGE1_USR_FROM="${RKT_STAGE1_USR_FROM:-coreos}"
 case "${RKT_STAGE1_USR_FROM}" in
-	coreos|src)
+	none)
+		echo "stage1 build disabled"
+		;;
+	coreos)
+		echo "stage1 will reuse PXE image"
+		;;
+	src)
+		echo "stage1 will be built from sources"
+		RKT_STAGE1_SYSTEMD_SRC="${RKT_STAGE1_SYSTEMD_VER:-git://anongit.freedesktop.org/systemd/systemd}"
+		export RKT_STAGE1_SYSTEMD_SRC
```

Comment:
You could put it in one line: `export ENV_VAR=value`. We already assumed that we need bash to run this script.